src/routes/projects.ts

```diff
@@ -17,16 +23,37 @@ router.get('/', async (req, res, next) => {
   }
 });
 
+// Returns all build data needed for all branches for a single project
+router.get('/:snootyProject/documents', async (req, res, next) => {
+  const { snootyProject } = req.params;
+
+  const { updated } = req.query;
```

Comment:
I know this isn't consistent with the other existing endpoints that use `../documents/updated/1234`, but using a query param is probably neater and more conventional (`../documents?updated=1234`). 

If this seems reasonable, I can create a ticket to change the other endpoints to follow this convention. Otherwise, I can split this out into 2 endpoints.

Reply:
Great. I can file a ticket for this